amgut/handlers/auth_handlers.py

```diff
@@ -42,7 +42,7 @@ def post(self):
             ag_login_id, skid, kitinfo['password'],
             kitinfo['swabs_per_kit'], kitinfo['verification_code'],
             printresults)
-        if success == -1:
+        if success != 1:
```

Comment:
Future-proofing mostly. I agree the method is concerning, but cleanup of the ag_data_access methods should probably happen as another pull request.